http/server.go

```diff
@@ -56,7 +56,7 @@ type ServerOptions struct {
 // DefaultOpts returns the default options for the server.
 func DefaultServerOptions() *ServerOptions {
 	return &ServerOptions{
-		Address: "127.0.0.1:9181",
+		Address: "http://127.0.0.1:9181",
```

Comment:
thought: I'm surprised that this doesn't cause problems. It's probably because the listener ignores the scheme. The scheme will actually be determined by whether TLS has been requested or not. I would prefer not adding it here.

Reply:
Understood. I have made this change as requested.